docs/use-signalr-service.md

```diff
@@ -92,6 +92,14 @@ By default, all claims from `HttpContext.User` of the negotiate request will be
 They can be accessed at [`Hub.Context.User`](https://github.com/aspnet/SignalR/blob/release/2.2/src/Microsoft.AspNetCore.SignalR.Core/HubCallerContext.cs#L29).
 - Normally you should leave this option as is. Make sure you understand what will happen before customizing it.
 
+#### `ServerStickyMode`
+- Default value is `Disabled`.
+- This option specifies the mode for **server sticky**. When the client is always routed to the server which it first `/negotiate` with, we call it **server sticky**. This options can be useful when you want to store some client context when `/negotaite` for `hub` to use.
+- In distributed scenarios, there can be multiple app servers connecting to one Azure SignalR instance. Inside one Azure SignalR instance, there are many running pods, each of them take some server connections from some app server. When the client comes in, Azure SignalR chooses the server connection for the client traffic to route to based on the routing strategy. 
```

Comment:
Since preferred is unpredictable, in which case this will more useful than disabled?

Reply:
should be preferred rather than prefered.